tmt/steps/prepare/distgit.py

```diff
@@ -19,6 +19,11 @@
     import tmt.steps.discover
     import tmt.steps.prepare.install
 
+#: Application of patches by ``rpm-build`` command invoked by
+#: :ref:`discover/fmf</plugins/discover/fmf>` plugin when
+#: ``dist-git-source`` key is used.
+PHASE_ORDER_RPMBUILD_BP = 30
```

Comment:
The default was previously `60` (see below). Why the value has been updated to `30`?

Reply:
A copy & paste issue, it's correct in docs. Fixed in d4f6cbab.

Reply:
Good!